src/createServer.js

```diff
@@ -1,11 +1,242 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const cors = require('cors');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
+  const app = express();
+
+  let users = [];
+  let expenses = [];
+
+  app.use(cors());
+
+  app.get('/users', (req, res) => {
+    if (!users) {
+      res.statusCode = 200;
+
```

Comment:
I guess it's unnecessary to do this check, if your users is empty we can just turn our array back.